### PR TITLE
consume G6G1 pedestals from HLT tag

### PR DESCRIPTION
**Pull request by argiro**
Modify the Ecal pedestal PCL workflow so that G6 and G1 pedestals are copied from the HLT label. The HLT will always contain the most updated measurement of G6 and G1 peds.

### CODE REVIEW

**Comment by arunhep** [comment]
please test

**Comment by arunhep** [comment]
+1

File: Calibration/EcalCalibAlgos/src/ECALpedestalPCLHarvester.cc [hunk 1]
```diff
@@ -156,6 +165,15 @@ bool ECALpedestalPCLHarvester::checkStatusCode(const DetId& id){
     return true;
 }
 
+bool ECALpedestalPCLHarvester::isGood(const DetId& id){
+    
+    EcalChannelStatusMap::const_iterator dbstatusPtr;
+    dbstatusPtr = channelStatus_->getMap().find(id.rawId());
+    EcalChannelStatusCode::Code  dbstatus = dbstatusPtr->getStatusCode();
```
Comment (davidlange6) [inline Calibration/EcalCalibAlgos/src/ECALpedestalPCLHarvester.cc]: hi @argiro - please handle the case of the rawId not being in the map

**Comment by argiro** [comment]
Hi @davidlange6 , I've added treatment of the case

**Comment by davidlange6** [comment]
please test

**Comment by arunhep** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**